scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -259,9 +260,19 @@ def _setup_main_tab(self):
         self.data_processor_table.processingFinished.connect(self._processing_finished)
         self.data_processor_widget_layout.addWidget(self.data_processor_table)
         self.data_processor_table.dataChanged.connect(self._data_changed)
-
         self.data_processor_table.instrumentHasChanged.connect(self._handle_instrument_change)
 
+    def _setup_main_tab(self):
+        # --------------------------------------------------------------------------------------------------------------
+        # Header setup
+        # --------------------------------------------------------------------------------------------------------------
```

Comment:
Are these big header comments necessary now that `setup_layout` has been refactored a bit? I would probably argue that the method name should be descriptive enough without, especially if they're this short - since `_setup_main_tab` now appears to set up the header, it should probably be renamed as such

Reply:
I think you are right about this header comment not really needed anymore in a 4 line function. I actually prefer setup_main_tab as a name though.